pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -459,7 +511,12 @@ func (l *listUsersQuery) expandExclusion(
 		}
 	}
 
-	return errs
+	finalErr := errs
```

Comment:
**Question:** Why the distinction of `finalErr`? If possible, I'd prefer to just keep as `errs`.

Reply:
My bad, yes. This was an unfounded fear related to https://100go.co/#returning-a-nil-receiver-45